BuildAndTest.cmd

```diff
@@ -57,17 +63,9 @@ echo         public const string Version = AssemblyVersion + Prerelease;
 echo     }                                                                          >> %DRV_VERSION_CONSTANTS%
 echo  }                                                                             >> %DRV_VERSION_CONSTANTS%
 
-if NOT exist "GeneratedKey.snk" (
-sn -k GeneratedKey.snk
-)
-
 @REM Build all code
 %~dp0.nuget\NuGet.exe restore src\Everything.sln -ConfigFile .nuget\NuGet.Config
```

Comment:
Hmm, I guess the comment on line 66 is required because it refers to the msbuild call on line 69. So just delete the nuget.exe call on line 67.